electroncash_gui/qt/main_window.py

```diff
@@ -2084,6 +2129,8 @@ def do_send(self, preview = False):
         if not self._chk_no_segwit_suspects():
             return
 
+        self._chk_no_legacy_address()
```

Comment:
Nit: The name `_chk_no_legacy_address` makes me think maybe that this function throws if legacy or does something that enforces the predicate `"There shall be no legacy addresses beyond this point"`.  

That is not the case.  I feel like renaming this function to `_warn_if_legacy_address` may be more apt / clear. Also renaming it indicates that the function throws up a warning, not just does a check.

Reply:
Renamed.